### DISCUSSION TITLE
I want to ask what is the maximum resolution it can train, what is the maximum resolution it can inference, of any model and trick?

### DISCUSSION THREAD

**Discussion by foreverpiano** [huggingface/diffusers#8586] [General]
Is there any doc that I can find the result?

**Comment by foreverpiano** [comment]
@asomoza @vladmandic @AmericanPresidentJimmyCarter  thanks for you help.

**Comment by asomoza** [comment]
You're asking a too broad question, there's a lot of models with a lot of resolutions, so you'll need to be more specific. Also you can mostly find this information reading the paper of each one.

There's no trick to this, it all depends on the architecture and on what resolutions the base model was trained.

Most people overcome this restriction with what is commonly known as "hi-res fix" which is mostly generating at a lower resolution, upscaling the image, dividing it in tiles and then doing a second pass with img2img.

**Comment by vladmandic** [comment]
what @asomoza said is typical and most common
additional tricks would be to use hidiffusion, resadapter or kohya-hires-fix (i'd list them in that order of preference) to reduce model hallucinating when generating on resolutions higher than it was trained on.

regarding maximum training resolution? no such thing other than your vram limitations - but if model doesn't have any base knowledge in that resolution, that training will be long and painful and results are questionable. so for any model, look up what are all the buckets it was originally trained on (its never just base resolution) - that's why its called fine-tuning, not training-from-scratch.
